### PR TITLE
Cortex xdr incident handling enhancement

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9874)

## Description
Cortex XDR incident handling v3:
Updated the conditional task that determines whether to continue with the investigation and response.
Cortex XDR Alerts Handling v2:
Added a context key for unhandled alerts.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

File: Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling_v2.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ tasks:
         "position": {
           "x": 480,
           "y": 70
-        }
+        }ƒ
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling_v2.yml:30]: Remove the unnecessary char

File: Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_6_5.yml [hunk 2]
```diff
@@ -1648,30 +1648,28 @@ tasks:
     task:
       id: 3ba84f9e-aaf6-4bd3-8d22-cfc00499e075
       version: -1
-      name: Should continue with the playbook's investigation and response?
+      name: Whether to continue with the investigation and response?
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_6_5.yml:1651]: I think we can rename the task to something like "check for unhandled alert" / "Check whether all alerts handled?" / etc

File: Packs/CortexXDR/ReleaseNotes/6_1_24.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+
+##### Cortex XDR incident handling v3
+
+Updated the conditional task that determines whether to continue with the investigation and response.
```
Comment (altmannyarden) [inline Packs/CortexXDR/ReleaseNotes/6_1_24.md:6]: if found unhandled alerts... something like that

File: playbook-Cortex_XDR_Alerts_Handling_v2_copy_1.yml [hunk 4]
```diff
@@ -0,0 +1,1888 @@
+id: Cortex XDR Alerts Handling v2_1
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: Cortex XDR Alerts Handling v2_1
```
Comment (altmannyarden) [inline playbook-Cortex_XDR_Alerts_Handling_v2_copy_1.yml:5]: what is that playbook? :)

File: Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_6_5.yml (hunk 2 above)
Comment (OmriItzhak) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_6_5.yml]: Agreed. changed to "Check whether there are any unhandled alerts?"

File: playbook-Cortex_XDR_Alerts_Handling_v2_copy_1.yml (hunk 4 above)
Comment (OmriItzhak) [reply playbook-Cortex_XDR_Alerts_Handling_v2_copy_1.yml]: Removed. Accidentally pushed to the PR.

File: Packs/CortexXDR/ReleaseNotes/6_1_24.md (hunk 3 above)
Comment (OmriItzhak) [reply Packs/CortexXDR/ReleaseNotes/6_1_24.md]: changed to "Updated the conditional task that determines whether to continue with the investigation and response to any unhandled alerts if they are found."

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**